fe/src/pages/user/Profile/Edit/ProfileEdit.jsx

```diff
@@ -20,37 +27,36 @@ function ProfileEdit() {
 
   return (
     <Fragment>
-      <Header />
-      <div className="container card p-2">
-        <div className="row gx-2">
-          <div className="col-lg-4 col-md-4 mb-2">
-            <div className="p-3 card bg-light">
-              <div className="d-flex justify-content-center">
-                <img
-                  className="mx-auto rounded-circle"
-                  style={{ width: 200, height: 200 }}
-                  src={currentUser.avatar ?? "/images/default_image.jpg"}
-                  alt="avatar"
+      <Container style={"card p-2"}>
```

Comment:
- you use curly braces if there are any additional props or statements you want to inline.
```suggestion
      <Container style="card p-2">
```

Reply:
- i still see same issue above.